tests/integration/targets/aws_kms/tasks/main.yml

```diff
@@ -32,6 +32,26 @@
         filters:
           alias: "{{ resource_prefix }}-kms"
 
+    - name: create a key in check mode
+      check_mode: yes
+      aws_kms:
+        alias: "{{ resource_prefix }}-kms-check"
+        tags:
+          Hello: World
+        state: present
+        enabled: yes
+
```

Comment:
Needs an `assert` task to verify that the module correctly returned 'changed'.

Reply:
Addressed.